cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/QueueAdmin.java

```diff
@@ -102,11 +102,4 @@
    */
   void drop(String name) throws Exception;
 
-  /**
-   * Performs update of entity.
-   *
-   * @param name Name of the entity to update
-   * @throws Exception if update fails
-   */
-  void upgrade(String name, Properties properties) throws Exception;
```

Comment:
There's an no parameter `upgrade()`upgrade method, which upgrades all tables that the queueAdmin is responsible for.